tools/github_actions/extract_release_notes.py

```diff
@@ -0,0 +1,40 @@
+import sys
+
+
+def extract_section(filename, heading):
+    with open(filename, "r") as file:
+        lines = file.readlines()
+
+    start_line, end_line = None, None
+
+    for i, line in enumerate(lines):
+        if line.startswith("# "):
+            current_heading = line.strip("#").replace(":", "").strip()
+            if current_heading == heading:
+                start_line = i
+            elif start_line is not None:
+                end_line = i
+                break
+
+    if start_line is not None:
```

Comment:
```suggestion
    ```suggestion
    if start_line is None:
        return None

    end_line = end_line or len(lines)
    section = "".join(lines[start_line + 1 : end_line]).strip()
    return section
```
```